types/src/transaction/transaction_v1/errors_v1.rs

```diff
@@ -10,48 +10,50 @@ use std::error::Error as StdError;
 use datasize::DataSize;
 use serde::Serialize;
 
+#[cfg(doc)]
+use super::TransactionV1;
 use crate::{crypto, TimeDiff, Timestamp};
 
-/// Returned when a `Transaction` fails validation.
+/// Returned when a [`TransactionV1`] fails validation.
```

Comment:
Wouldn't

```
[`super::TransactionV1`](TransactionV1)
```
work  (or something like that)? We could avoid `#[cfg(doc)]`. It's not a big deal, just wondering

Reply:
Well it'd be:
```rust
[`TransactionV1`](super::TransactionV1)
```
but yes it'd be equivalent.  I've just been preferring the imports approach throughout.